azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/fragment/calling/participant/grid/cell/ParticipantGridCellVideoView.kt

```diff
@@ -24,12 +25,13 @@ import kotlinx.coroutines.launch
 internal class ParticipantGridCellVideoView(
```

Comment:
do we care about seeing/using this View in Studio's Layout Editor?
If so, can we add the missing View constructors, if not can we add the annotation to suppress compiler warning